apps/catalog/catalog/src/app/dashboard/sales/shell/view/view.component.ts

```diff
@@ -20,16 +20,16 @@ export class SaleViewComponent {
   orgId = this.query.getActiveId();
 
   constructor(
-    private contractService: ContractService,
+    private negotiationService: NegotiationService,
     private snackbar: MatSnackBar,
     private shell: SaleShellComponent,
     private dialog: MatDialog,
     private query: OrganizationQuery
   ) { }
 
-  accept(id: string, movie: Movie) {
+  accept(contractId: string, negotiationId: string, movie: Movie) {
```

Comment:
for coherence switch negotiationId & contractId : 
```suggestion
  accept(negotiationId: string, contractId: string, movie: Movie) {
```